src/models/enums/index.ts

```diff
@@ -0,0 +1 @@
+export * from './viewportType';

```

Comment:
The package should be organized by feature, not by type. This should be defined in src\commands\viewport.ts

Reply:
I agree 👍